### DISCUSSION TITLE
download playlists

### DISCUSSION THREAD

**Discussion by githubuser872349** [FreeTubeApp/FreeTube#2222] [Q&A]
Is there a way to download whole playlist with newpipe instead of downloading every video/audio on its own?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
There is no way to do that as of now. See open issue [here](https://github.com/FreeTubeApp/FreeTube/issues/643)

**Reply by githubuser872349** [reply]
Thank you for the quick answer.
i really hope this feature will be added soon.
